comparators/disputed-border-tag-changed.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+module.exports = disputedBorderTagChanged;
+
+function disputedBorderTagChanged(newVersion, oldVersion, callback) {
+  var result = {};
+  result['result:disputed_border_tag'] = {};
+
+  if (!newVersion && !oldVersion) {
+  // None of old version or new Version present
+    return callback(null, {});
+  }
+  if (newVersion && oldVersion) {
+  // Both new Version and old Version are present, which indicates feature has been modified
+  /*
+    Comparing the tags
+    Creating a result object like following:
+    result['result:comparator_name'][parameter] = value;
+  */
+    if (oldVersion.properties && oldVersion.properties['disputed']) {
+      if (oldVersion.properties['disputed'] !== null) {
+        result['result:disputed_border_tag']['modified'] = true;
```

Comment:
Sorry for the confusion in boiler plate instructions. But parameter is not required anymore with the result as we have only one suspicion reason per comparator. So it can be `result['result:disputed_border_tag'] = true;`